indico/modules/categories/controllers/management.py

```diff
@@ -325,18 +319,11 @@ def _checkParams(self):
                 raise BadRequest(_("Cannot move a category in a descendant of itself."))
 
     def _process(self):
-        if self.target_category is None:
-            return jsonify_template('categories/management/move_category_contents.html', categories=self.subcategories,
-                                    submit_url=url_for('.move_subcategories', self.category),
-                                    category_ids=[x.id for x in self.subcategories])
-        if 'confirmed' in request.form:
-            for subcategory in self.subcategories:
-                move_category(subcategory, self.target_category)
-            flash(ngettext('{} category moved in "{}".', '{} categories moved in "{}".', len(self.subcategories))
-                  .format(len(self.subcategories), self.target_category.title), 'success')
-            return jsonify_data(flash=False)
-        return jsonify_template('categories/management/move_categories.html', categories=self.subcategories,
-                                category_ids=[x.id for x in self.subcategories], target_category=self.target_category)
+        for subcategory in self.subcategories:
+            move_category(subcategory, self.target_category)
+        flash(ngettext('{} category moved to "{}".', '{} categories moved to "{}".', len(self.subcategories))
+              .format(len(self.subcategories), self.target_category.title), 'success')
```

Comment:
always use positional or named format arguments for i18n if there's more than once placeholder!